astropy/tests/pytest_plugins.py

```diff
@@ -147,8 +147,9 @@ def pytest_configure(config):
                           allow_astropy_data=config.getoption('remote_data') == 'astropy')
 
     doctest_plugin = config.pluginmanager.getplugin('doctest')
-    if (doctest_plugin is None or config.option.doctestmodules or not
-            (config.getini('doctest_plus') or config.option.doctest_plus)):
+    if (doctest_plugin is None or config.option.doctestmodules
+            or not (config.getini('doctest_plus').lower() in ['true', 'enabled']
```

Comment:
Yes, it complained to me, too, but there is this change of policy that hasn't yet make its way to all the linters:

https://hg.python.org/peps/rev/3857909d7956
https://github.com/PyCQA/pycodestyle/issues/498